vocabularies/earth-science-data-category.ttl

```diff
@@ -359,15 +366,8 @@ esdc:geothermics-and-radiometrics a skos:Concept ;
     skos:exactMatch <http://purl.org/au-research/vocabulary/anzsrc-for/2008/040404> ;
     skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
     skos:notation "040404" ;
-    skos:prefLabel "Geothermics and Radiometrics"@en ;
-    skos:altLabel "Radiometrics"@en .
-
-esdc:geothermal-energy a skos:Concept ;
-    skos:broader esdc:geothermics-and-radiometrics ;
-    skos:definition "The study of heatflow in the Earth's upper crust in the context of energy generation, and methods, materials and operations used in extraction of energy from geothermal sources."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
-    skos:altLabel "Geothermal"@en ;
-    skos:prefLabel "Geothermal Energy"@en .
+    skos:prefLabel "Radiometrics"@en ;

```

Comment:
I preferred the other terms list.  But in terms of radiometric they way we use it maybe it should be separate as it is not about Geothermics and Radiometrics and heat flow, but surface radioisotopes

Reply:
[ANZSRC-FOR](https://vocabs.ands.org.au/viewById/20) classifies it as Geothermics and Radiometrics.
If they should be separate I suggest that we introduce them as narrower concepts so we still retain a direct mapping to the established standard.

As a preference I would really like to get this PR approved and start a new one while these edits are still timely and this change stacks up too many commits to easily follow the changes.

Reply:
Yeah i am happy as 80:20 to go,  But on a techincal point i think we are still wrong path for this element

Reply:
For ANZSCR-FOR this is why i think it is not suited to the ESDC as the data categories do not relate to the fields of research often